react-native-libraries.json

```diff
@@ -11630,5 +11630,25 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/pmndrs/zustand",
+    "npmPkg": "zustand",
+    "examples": [
+      "https://zustand-demo.pmnd.rs/",
+      "https://github.com/pmndrs/zustand/tree/main/examples/demo"
+    ],
+    "images": [
+      "https://github.com/pmndrs/zustand/raw/main/docs/bear.jpg"
+    ],
```

Comment:
```suggestion
```

We do not accept branding images:
* https://github.com/react-native-community/directory?tab=readme-ov-file#images